translations/base-fr.yaml

```diff
@@ -360,7 +382,10 @@ ingame:
         noShapesProduced: Aucune forme produite pour le moment.
 
         # Displays the shapes per minute, e.g. '523 / m'
-        shapesPerMinute: <shapes> ⁄ m
+        shapesDisplayUnits:
+            second: <shapes> / s
```

Comment:
Should be ` ⁄ ` (U+00A0 NO-BREAK SPACE and U+2044 FRACTION SLASH)

Reply:
I have tried to copy but I am not shure it is good.